lib/seed_base_items.rb

```diff
@@ -0,0 +1,19 @@
+def seed_base_items
```

Comment:
Top-level functions look kind of weird in a Rails app. Can this be wrapped in a module?

It worked in `rails_helper` because those kind of helper methods are fine if used specifically for test setup.

Reply:
added to module `seeds` in commit [94030f7](https://github.com/rubyforgood/human-essentials/pull/4665/commits/94030f75aefe8cfa6ecc4e6155da4c8e18bb630f)